frontend/src/App.tsx

```diff
@@ -70,27 +70,37 @@ const App = (): React.ReactElement => {
               <Route exact path={Routes.LOGIN_PAGE} component={Login} />
               <Route exact path={Routes.SIGNUP_PAGE} component={Signup} />
               <Route exact path={Routes.HOOKS_PAGE} component={HooksDemo} />
-              <PrivateRoute exact path={Routes.HOME_PAGE} component={Default} />
+              <PrivateRoute
+                exact
+                path={Routes.HOME_PAGE}
+                component={Default}
+                allowedRoles={["Administrator", "Facilitator", "Learner"]}
+              />
               <PrivateRoute
                 exact
                 path={Routes.CREATE_ENTITY_PAGE}
                 component={CreatePage}
+                allowedRoles={["Administrator", "Facilitator", "Learner"]}
               />
               <PrivateRoute
                 exact
                 path={Routes.UPDATE_ENTITY_PAGE}
                 component={UpdatePage}
+                allowedRoles={["Administrator", "Facilitator", "Learner"]}
               />
               <PrivateRoute
                 exact
                 path={Routes.DISPLAY_ENTITY_PAGE}
                 component={DisplayPage}
+                allowedRoles={["Administrator", "Facilitator", "Learner"]}
               />
               <PrivateRoute
                 exact
                 path={Routes.EDIT_TEAM_PAGE}
                 component={EditTeamInfoPage}
+                allowedRoles={["Administrator", "Facilitator", "Learner"]}
               />
+              <Route exact path={Routes.NOT_FOUND_PAGE} component={NotFound} />
```

Comment:
This line is redudant since ```<Route exact path="*" component={NotFound} />``` will match any path that hasn't been hit by a route above it.

Reply:
I noticed that when it's 404, it keeps the same url and adds a  * at the end. I added the specific 404 route so it gives more user feedback about the url, but maybe that's pretty obvious when the page literally says not found.

Reply:
Makes sense, if that's the case we could make it so that on any unknown page we give a descriptive URL. Good catch
```
 <Route exact path={Routes.NOT_FOUND_PAGE} component={NotFound} />
 <Redirect to={Routes.NOT_FOUND_PAGE} />
```